cmd/pack3d/main.go

```diff
@@ -58,19 +65,19 @@ func main() {
 	count := 1
 	ok := false
 
-
 	/*Loading frame size*/
-	for _, j := range os.Args[1:4]{
-		_dimension, err := strconv.ParseInt(j, 0, 0)
+	for _, j := range os.Args[1:5]{
```

Comment:
/* unpackes first 5 values, to set build_volume and spacing */ 

This is a very fragile way to unpack data.  We need to keep moving, but if we make more changes, we should use a real parameter unpacking tool.